_topic_maps/_topic_map_rosa.yml

```diff
@@ -1428,33 +1428,34 @@ Topics:
     File: ossm-reference-jaeger
   - Name: Uninstalling Service Mesh
     File: removing-ossm
-- Name: Service Mesh 1.x
```

Comment:
PEER REVIEWER: Service Mesh was in ROSA docs ^^. Hiding V1 as it is in Tech Preview.